debian/po/pt_BR.po

```diff
@@ -1664,8 +1664,8 @@ msgid "Collect logs and relevant system information into a tarball."
 msgstr "Coleta logs e outras informações relevantes do sistema em um tarball."
 
 #: ../../uaclient/messages/__init__.py:898
-msgid "tarball where the logs will be stored. (Defaults to ./ua_logs.tar.gz)"
-msgstr "tarball onde os logs serão guardados. (Valor padrão ./ua_logs.tar.gz)"
+msgid "tarball where the logs will be stored. (Defaults to ./pro_logs.tar.gz)"
+msgstr "tarball onde os logs serão guardados. (Valor padrão ./pro_logs.tar.gz)"
```

Comment:
This will need updating in the docs too. For example, https://canonical-ubuntu-pro-client.readthedocs-hosted.com/en/latest/howtoguides/how_to_collect_logs/?highlight=collect-logs mentions `ua_logs.tar.gz`.

Reply:
Already done in the `docs-devel` branch which will get published after releasing the package https://github.com/canonical/ubuntu-pro-client/blob/docs-devel/docs/howtoguides/how_to_collect_logs.rst